devel/conventions/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout:   default
+class:    markdown
+---
+
+Conventions
+===========
+
+TaskCluster is a very open platform, and can easily descend into anarchy if we are not careful to enforce some standard approaches and conventions.
```

Comment:
"Loosely typed" or " dynamic platform" also works...

Note, these conventions are just as much for us, as they are for customers... In fact they are mainly relevant to us and maybe power users..